site/docs/v1/tech/identity-providers/sony.adoc

```diff
@@ -35,7 +35,7 @@ link:https://partners.playstation.net/[Sony Playstation Network - Getting Starte
 
 == Create a Sony Playstation Network Identity Provider
 
-:identity_providerPlaystation:
+:identity_provider: Playstation
```

Comment:
Not sure where this is used, but should it be `Sony PlayStation Network` that is what our display name is in FusionAuth.

Reply:
Yeah, I just noticed it was messed up while I was looking it over as an example. I'll change it to Sony Playstation Network